packages/wranglerjs-compat-webpack-plugin/src/__tests__/helpers/install-wrangler.ts

```diff
@@ -26,7 +26,7 @@ export async function installWrangler1() {
 		data: { assets },
 	} = await octokit.request("GET /repos/{owner}/{repo}/releases/latest", {
 		owner: "cloudflare",
-		repo: "wrangler",
+		repo: "wrangler-legacy",
```

Comment:
Should this hold off on being merged until the repos have been renamed?

Reply:
yep! and also @caass and i have coordinated and i have some additional changes i want to include so converting this to draft 👍